src/App.js

```diff
@@ -8,36 +8,35 @@ const HowToUse = () => {
   return (
     <div className="subsection" id="howtouse">
       <h1>How To Use?</h1>
-      <br />
-      <br />
-      <br />
-      <br />
-      <br />
-      <br />
-      <br />
-      <br />
-      <br />
-      <br />
-      <br />
-      <br />
-      <br />
-      <br />
-      <br />
-      <br />
-      <br />
-      <br />
-      <br />
-      <br />
-      <br />
-      <br />
-      <br />
-      <br />
-      <br />
-      <br />
-      <br />
-      <br />
-      <br />
-      <br />
+      <div align="left" style= {{ padding:"5%" }}>
+        <h2>Creating your Handwritten Sample</h2>
+        <br />
+        <ol type="1">
+          <li> Take a printout of the sample form.</li> <br />
+          <li> Fill the form using the image below as a reference.</li> <br />
+          <li> Scan the filled form using a scanner, or Adobe Scan in your phone.</li> <br />
+          <li> Save the .jpg image in your system.</li> <br />
+        </ol>
+        Your form should look like this:
+        <br /> <br />
+        <img src="https://raw.githubusercontent.com/cod-ed/assets/handwrite/handwrite_filled_form.jpg" 
+          className="image-container"></img>
+        <br /> <br />
+
+        <h2>Creating your font</h2>
+        <br />
+        <ol type="1">
+          <li>Upload the scanned .jpg image in the form above.</li> <br />
```

Comment:
```suggestion
          <li>Upload the scanned JPEG image in above.</li> <br />
```

We can also link above to home.